src/components/Modal/FeedbackModal/styled.ts

```diff
@@ -0,0 +1,105 @@
+import styled from "styled-components";
+import FONT from "../../../styles/font";
+
+export const CloseIcon = styled.img<{ onClick: () => void }>`
+  position: absolute;
+  top: 16px;
+  right: 20px;
+  cursor: pointer;
+`;
+
+export const ModalContainer = styled.div`
```

Comment:
```suggestion
export const ModalContainer = styled.form`
```
내부적으로 textArea 태그도 있고 서버에 post 통신을 하기 때문에 form 태그를 사용하는 것도 시맨틱해서 좋을 것 같아요!